--- Makefile.orig	2013-10-31 14:39:14.543199769 +0100
+++ Makefile	2013-10-31 14:39:48.482555147 +0100
@@ -3,7 +3,7 @@
 COMPILER=gfortran
 ## Choose PDF: native,lhapdf
 ## LHAPDF package has to be installed separately
-PDF=native
+PDF=lhapdf
 #Choose Analysis: none, B-or-D, top
 ## default analysis may require FASTJET package, that has to be installed separately (see below)
 ANALYSIS=top
@@ -23,6 +23,7 @@
 OPT=-O2
 ## For debugging uncomment the following
 #DEBUG= -ggdb -pg
+LIBS=-lz
 endif
 
 ifeq ("$(COMPILER)","g77")
@@ -65,7 +66,7 @@
 ifeq ("$(PDF)","lhapdf")
 LHAPDF_CONFIG=lhapdf-config
 PDFPACK=lhapdfif.o
-LIBSLHAPDF= -Wl,-rpath,$(shell $(LHAPDF_CONFIG) --libdir)  -L$(shell $(LHAPDF_CONFIG) --libdir) -lLHAPDF
+LIBSLHAPDF= -lLHAPDF #-Wl,-rpath,$(shell $(LHAPDF_CONFIG) --libdir)  -L$(shell $(LHAPDF_CONFIG) --libdir) -lLHAPDF
 ifeq  ("$(STATIC)","-static") 
 ## If LHAPDF has been compiled with gfortran and you want to link it statically, you have to include
 ## libgfortran as well. The same holds for libstdc++. 
@@ -92,8 +93,8 @@
 else
 ifeq ("$(ANALYSIS)","top")
 FASTJET_CONFIG=$(shell which fastjet-config)
-LIBSFASTJET += $(shell $(FASTJET_CONFIG) --libs --plugins ) -lstdc++
-FJCXXFLAGS+= $(shell $(FASTJET_CONFIG) --cxxflags)
+LIBSFASTJET += -lstdc++ -lfastjet #$(shell $(FASTJET_CONFIG) --libs --plugins ) -lstdc++
+FJCXXFLAGS+= #$(shell $(FASTJET_CONFIG) --cxxflags)
 PWHGANAL=pwhg_analysis-top.o fastjetfortran.o
 else
 PWHGANAL=pwhg_analysis-dummy.o
